AirSkyBoat/AirSkyBoat
# Add Brown Magic to FFXI

**_I affirm:_**


## Please enter a player-facing description

Borwn Magic has been added to FFXI.  See Parike-Poranke in Lower Jeuno for more details.

## What does this pull request do? (Please be technical)

Adds Parike-Poranke's [Brown Magic](https://ffxiclopedia.fandom.com/wiki/Brown_Magic) functionality.
Based on a capture by Siknoz and some inferrence from the event decompile.

## Steps to test these changes

Go to Parike-Poranke

===Basics===
Talk to them w/o food on.
Talk to them w/ food on.
Trade a gyshal green w/ food on - have food removed.
Trade a gyshal green w/ food on again - have food removed, note the number of success have increased..
Trade gyshal green w/o food on - note that you get diseased, then diseased cured, and the number of failures increments

===Test skill ups===
!exec SetServerVariable("Parike_DigestiveSkill", 999)
eat food, trade gyshal greens - note that this is success 1000.  Note that you see a digestive magic skill rises 0.1 points message.

!exec SetServerVariable("Parike_ScienceSkill", 999)
dont eat food, trade gyshal greens - note that this is failure 1000.  Note that you see a for the sake of science magic skill rises 0.1 points message.

===Test gaining a level===
!exec SetServerVariable("Parike_DigestiveSkill", 9999)
eat food, trade gyshal greens - note that this is success 10000.  Note that you see a digestive magic skill rises 0.1 points message. Note that you also see that it gains a level.
Note that there is an add on to the CS where Parike says he is giving you a gift with a big animation.
Check the player - they should have a new title.

!exec SetServerVariable("Parike_ScienceSkill", 9999)
dont eat food, trade gyshal greens - note that this is failure 10000.  Note that you see a for the sake of science magic skill rises 0.1 points message. Note that you also see that it gains a level.
Note there is an add on to the CS where Parike talks about discovering a new spell like flee, then forgetting it.  he casts a haste like spell effect on the player.  Player gains flee (30s) and player gains a new title.

## Special Deployment Considerations

None, although SE used to reset his counters on maintence.
